lib/ui/page/home/page/chat/widget/chat_forward.dart

```diff
@@ -440,7 +440,7 @@ class _ChatForwardWidgetState extends State<ChatForwardWidget> {
                     media.first,
                     galleries,
                     key: _galleryKeys[msg.id]?.firstOrNull,
-                    onGallery: widget.onGallery,
+                    onGallery: menu ? null : widget.onGallery,
```

Comment:
You forgot about `ChatForward`s - those types of `ChatItem`s are displayed via separate widget due to complexity of forwards.